activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+*   `order` now supports Postgres JSON operators.
+
+    `order` raises `ActiveRecord::UnknownAttributeReference` when it can't identify the syntax of
+     the given string. Now it identifies the `->>` operator in Postgres instead of raising an error.
+
+    Old syntax:
+    ```ruby
+        Event.order(Arel.sql("payload->>'kind'"))
+    ```
+
+    New syntax:
+    ```ruby
+        Event.order("payload->'kind'")
```

Comment:
Should this be `Event.order("payload->>'kind'")`?

Reply:
Thanks for catching that!